pages/resources/user.mdx

```diff
@@ -719,7 +719,7 @@ Returns a list of active [private channel](/resources/channel#channel-object) ob
   Create Private Channel
 </RouteHeader>
 
-One recipient creates or returns an existing [DM channel](/resources/channel#channel-object), no or multiple recipients create a [group DM channel](/resources/channel#channel-object). Returns a [private channel](/resources/channel#channel-object) object. Fires a [Channel Create](/topics/gateway-events#channel-create) Gateway event.
+One recipient creates or returns an existing [DM channel](/resources/channel#channel-object), no or multiple recipients create a [group DM channel](/resources/channel#channel-object). Returns a [private channel](/resources/channel#channel-object) object. Fires a [Channel Create](/topics/gateway-events#channel-create) Gateway event. (If multiple channels with a single recipient exist, the channel with the largest ID is returned)
```

Comment:
```suggestion
One recipient creates or returns an existing [DM channel](/resources/channel#channel-object), no or multiple recipients create a [group DM channel](/resources/channel#channel-object). Returns a [private channel](/resources/channel#channel-object) object. Fires a [Channel Create](/topics/gateway-events#channel-create) Gateway event. 

If multiple channels with a single recipient exist, the most recent one is returned.
```

Reply:
"Recent is meh here since were talking about literal miliseconds, largest ID is more clear cut

Reply:
Largest ID is unnecessarily verbose; recent is more in-line with how [snowflakes work](https://discord-userdoccers.vercel.app/reference#encryption).

Reply:
Docs should be as verbose as u can get :)

Reply:
Verbosity is appreciated when it adds additional context. Largest ID does not offer any extra information; it offers less information when taking the structure of a snowflake into account.